chips/earlgrey/src/chip.rs

```diff
@@ -396,11 +397,8 @@ pub extern "C" fn _start_trap_vectored() -> ! {
             j _start_trap
             j _start_trap
             j _start_trap
-        "
-        :
-        :
-        :
-        : "volatile");
+            "
+        );
         unreachable_unchecked()
```

Comment:
This should use the `noreturn` option -- I actually made this change and a couple more to this file in #2319 , which I think encompasses everything in this PR.